vmdb/app/views/layouts/_dhtmlxgrid.html.haml

```diff
@@ -19,7 +19,8 @@
   - unless @parent.nil?
     var miq_parent_id = "#{@parent.id}";
     var miq_parent_class = "#{@parent.class.base_class.to_s.underscore}";
-    if (miq_parent_class == "policy_set") miq_parent_class = "policy";
+    if (miq_parent_class == "policy_set") { miq_parent_class = "policy"; }
+    else if ((miq_parent_class == "ext_management_system") && ("#{@layout}" == "ems_infra" || "#{@layout}" == "ems_cloud")) { miq_parent_class = "#{@layout}"; }
```

Comment:
@mzazrivec we also need to handle ems_container controller as well, i am not sure why you have "#{@layout} == "xx", instead of @layout == "xx" or can we just do:
 else if ((miq_parent_class == "ext_management_system")  miq_parent_class = @layout; }